packages/form-js-editor/src/render/components/properties-panel/groups/ValuesGroups.js

```diff
@@ -0,0 +1,52 @@
+import { ValuesSourceSelectorEntry, StaticValuesSourceEntry, InputKeyValuesSourceEntry, getValuesSource, VALUES_SOURCES } from '../entries';
+
+import { Group, ListGroup } from '@bpmn-io/properties-panel';
+
+import {
+  VALUES_INPUTS
+} from '../Util';
+
+export default function ValuesGroups(field, editField) {
```

Comment:
Can you explain why we have a single file for all the groups? It's now harder to read as the logic for multiple groups is handled in a single file, not following the same structure as beforehand.

Shouldn't the previous `ValuesGroup` still work (maybe with some different labels and some configuration)?

Reply:
I've done this simply because the groups are very tightly bound. Looking at this, you can instantly tell that there's a logical link between the three groups in the code. You see the conditional group choosing code.

If you we separated the code out into multiple groups we'd have:

ValuesSourceGroup => always visible
StaticValuesGroup => visible if valuesSource == static
DynamicValuesGroup => visible if valuesSource == dynamic

I think that at a glance, grouping those in a single file tells the story of what these are about better.